lib/game.rb

```diff
@@ -1,14 +1,14 @@
 class Game
   attr_reader :player1, :player2, :game_board
   attr_accessor :player_choice, :not_the_end
-
+  @@counter = 0
   def initialize(player1, player2, player_choice, game_board)
     @player1 = player1
     @player2 = player2
     @player_choice = player_choice.player_choice
     @game_board = game_board
     @not_the_end = true
-    @@counter = 0
+    # @@counter = 0
```

Comment:
- [ ]  Please remove this commented piece of code, if you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) .